tensorflow/compiler/xla/service/gpu/gemm_algorithm_picker.cc

```diff
@@ -47,6 +47,34 @@ using tensorflow::AutotuneResult;
 
 namespace {
 
+StatusOr<se::cuda::BlasLt::Epilogue> AsBlasLtEpilogue(
+    GemmBackendConfig_Epilogue epilogue) {
+  switch (epilogue) {
+    case GemmBackendConfig::DEFAULT:
+      return se::cuda::BlasLt::Epilogue::kDefault;
+    case GemmBackendConfig::RELU:
+      return se::cuda::BlasLt::Epilogue::kReLU;
+    case GemmBackendConfig::BIAS:
+      return se::cuda::BlasLt::Epilogue::kBias;
+    case GemmBackendConfig::BIASRELU:
+      return se::cuda::BlasLt::Epilogue::kBiasThenReLU;
+    case GemmBackendConfig::GELU:
+#if CUDA_VERSION >= 11040
```

Comment:
Does this have to be an ifdef vs a plain `if`?  i.e. `se::cuda::BlasLt::Epilogue::kGeLU` is defined unconditionally, so we should be able to use it unconditionally too, no?

We prefer plain `if` to `ifdef` where possible so that we're compiling fewer variants of the code.